### PR TITLE
Enable E2E tests for release-4.11 branch

### PR DESCRIPTION
**Pull request by vbnrh**
Signed-off-by: Vineet Badrinath <vineetbnath@gmail.com>

### CODE REVIEW

**Review by rexagod — APPROVED** [review]
The PR makes sense, however I'd suggest using `make new-repo` alongwith your usual targets (`make jobs`, `make ci-operator`, etc.) to avoid formatting errors.

File: ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.11-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,182 @@
+presubmits:
+  red-hat-storage/odf-console:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^release-4\.11$
+    - ^release-4\.11-
```
Comment (rexagod) [inline ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.11-presubmits.yaml:7]: Compatibility jobs should to target 4.12.

**Review by rexagod — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-master-postsubmits.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+postsubmits:
```
Comment (rexagod) [inline ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-master-postsubmits.yaml:1]: Do we need a [postsubmits](https://docs.ci.openshift.org/docs/architecture/ci-operator/#post-submit-tests) file?

**Review by rexagod — CHANGES_REQUESTED** [review]

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml [hunk 3]
```diff
@@ -15,6 +15,9 @@ images:
       as:
       - node:15
   to: odf-console
+promotion:
+  name: "4.12"
+  namespace: ocp
```
Comment (rexagod) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:20]: Do we need this? Shouldn't this change be in [`releases.integration`](https://docs.ci.openshift.org/docs/getting-started/examples/#how-do-i-use-an-image-from-another-repo-in-my-repos-tests) instead?

**Review by rexagod — CHANGES_REQUESTED** [review]

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml (hunk 3 above)
Comment (rexagod) [reply ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml:20]: Like so: https://github.com/openshift/release/pull/32612/files.

cc @SanjalKatiyar (your PR can be closed if @vbnrh makes this change in this PR itself).

**Comment by bipuladh** [comment]
/retitle Enable E2E tests for release-4.11 branch

File: ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.11-presubmits.yaml (hunk 1 above)
Comment (vbnrh) [reply ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.11-presubmits.yaml:7]: This PR only enables for 4.11 branch. The rehearses are only ran for 4.11 branch

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml (hunk 3 above)
Comment (vbnrh) [reply ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-master.yaml]: This is not for changing ocp version to 4.12.
I had added them for automated branching but can be removed for now.
https://docs.ci.openshift.org/docs/architecture/branching/#how-do-i-opt-my-repository-into-automated-branching

File: ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.11-presubmits.yaml (hunk 1 above)
Comment (rexagod) [reply ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.11-presubmits.yaml:7]: Oh, that's cool then! 👍🏼

**Pull request merged**